### DISCUSSION TITLE
yolov9 gelan support in ultralytics

### DISCUSSION THREAD

**Discussion by ShivankYadav** [ultralytics/ultralytics#11039] [Q&A] [answered]
@glenn-jocher 
As yolov9 models now got support in ultralytics itself. is it possible to reperemeterize those ultralytics's v9 models to gelan models?
and when will gelen models be avilable in ultralytics for training or auto reperameterization of v9 models into gelan?
Regards.

**Comment by glenn-jocher** [comment, marked as answer]
Hello! 🚀

Thanks for reaching out with your question. As of now, there are no official announcements or features regarding YOLOv9 GELAN models support or auto-reparameterization into GELAN within Ultralytics YOLO. We're continuously working on integrating more advancements into our toolkit, so keep an eye on our [GitHub](https://github.com/ultralytics/ultralytics) for updates! 

Meanwhile, you're welcome to experiment and contribute any findings you might have regarding reparameterizing YOLO models to GELAN. We appreciate input from the community to further enhance our models and offerings.

Best regards!
